_extras/guide.md

```diff
@@ -2,7 +2,13 @@
 title: "Instructor Notes"
 ---
 
-## Working with skimage
+## Working with imageio and skimage
+
+* `imageio.v3` allows to load images in different modes by passing the `mode=` argument to `imread()`. Depending on the image file and mode, the `dtype` of the resulting Numpy array can be different (e.g., `dtype('uint8')` or `dtype('float64')`. In the lesson, `skimage.util.img_as_ubyte()` and `skimage.util.img_as_float()` are used to convert the data type when necessary.
```

Comment:
This is a good candidate for relocating to an inline Instructor Notes (perhaps during the Instructor Notes Drive in May!) when the lesson has transitioned to the Workbench.